include/nuttx/audio/i2s.h

```diff
@@ -286,6 +309,11 @@ struct i2s_ops_s
                             FAR void *arg,
                             uint32_t timeout);
 
+  /* Master Clock methods */
+
+  CODE uint32_t (*i2s_mclkfrequency)(FAR struct i2s_dev_s *dev,
```

Comment:
Are the all i2c_ops_s users updated to handle insertion of the new method accordingly?

Reply:
The call of i2s_mclkfrequency already check the pointer isn't NULL, so it isn't a problem.